sync_radar_data_globus.py

```diff
@@ -174,32 +183,37 @@ def synchronize(self):
                 pass
             print("Listing path: {path} on endpoint: {endpoint} with pattern: {pattern}".format(
                 path=listing_path, endpoint=self.mirror_uuid, pattern=listing_pattern))
+            print("Note: This can take several minutes")
             listing_succeeded = False
-            operation_ls_retries = 10 
+
+            # The number of retries required to reliably succeed, on cedar's endpoint, to get
+            # around the timeout issue.
+            operation_ls_retries = 15
             attempts = 0
             listing = []
             listing_times = []
 
             while attempts < operation_ls_retries:
                 try:
                     listing_times.append(time.time())
-                    print("Attempt: {}".format(attempts)),
+                    if PYTHON3:
```

Comment:
Any reason to change the printing format method? I thought `.format` works in 2.7?

Reply:
Didn't change, I just decided to remove the print statement with number of attempts.